rawconn_lt_1.12.go

```diff
@@ -0,0 +1,29 @@
+//+build !go1.12
+
+package netlink
+
+import (
+	"os"
+	"syscall"
+)
+
+func newRawConn(fd *os.File) (syscall.RawConn, error) {
+	return &rawConn{fd: fd.Fd()}, nil
+}
+
+var _ syscall.RawConn = &rawConn{}
+
+// A rawConn is a syscall.RawConn.
+type rawConn struct {
+	fd uintptr
+}
+
+func (rc *rawConn) Control(f func(fd uintptr)) error {
+	f(rc.fd)
+	return nil
+}
+
+// TODO(mdlayher): implement Read and Write?
```

Comment:
Super nit: but future note for me just to drop this TODO; it's better to just not support it on <1.12.